src/main/java/com/dreammaster/gthandler/recipes/MixerRecipes.java

```diff
@@ -490,6 +490,20 @@ public void run() {
                 .itemOutputs(GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 4L)).duration(10 * SECONDS)
                 .eut(8).addTo(mixerRecipes);
 
+        GTValues.RA.stdBuilder()
+                .itemInputs(
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Tin, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Silver, 32L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Platinum, 32L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Thaumium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.EnderPearl, 64L),
+                        GTUtility.getIntegratedCircuit(12))
+                .itemOutputs(GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L),
+                        GTOreDictUnificator.get(OrePrefixes.dust, Materials.Enderium, 64L)).duration(16 * MINUTES)
+                .eut(TierEU.RECIPE_MV).noOptimize().addTo(mixerRecipes);
```

Comment:
no sorry but not mv. To skip enderium bas it must be a few tier higher

Reply:
why github show this is outdated? its not

Reply:
Im in the middle of fixing it right now, probably gonna close this PR as the enderium recipe will be added in the ABS and the lapotron recipe removed